trusted-types/block-string-assignment-to-Element-setAttributeNode.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<meta http-equiv="Content-Security-Policy" content="require-trusted-types-for 'script';">
+<title>trusted-types (TT): setAttributeNode with node from non-TT realm</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id="nonSVGTestElements">
+    <iframe srcdoc="v"></iframe>
+    <embed src="v">
+    <script src="v"></script>
+    <object data="v"></object>
+    <object codebase="v"></object>
+</div>
+<svg id="svgTestElements">
+    <script href="v"></script>
+    <script xlink:href="v"></script>
+</svg>
+<script>
+    const passThroughPolicy = trustedTypes.createPolicy("passThrough", { createHTML: s => s });
+
+    function runTest(aTestElement) {
+        const testAttr = aTestElement.attributes[0];
+
+        async_test(t => {
+            const sourceFrame = document.createElement("iframe");
+
+            sourceFrame.srcdoc = passThroughPolicy.createHTML(
+            `<!DOCTYPE html>
+            <head>
+            <meta charset="utf-8">
+            </head>
+            <body><div ` + testAttr.name + `="` + testAttr.value + `"></div>
+            doc without TT CSP.
+            </body>`);
+
+            t.add_cleanup(() => {
+                sourceFrame.remove();
+            });
+
+            sourceFrame.addEventListener("load", t.step_func_done(() => {
+                // A window is a global object which has 1-to-1 mapping to a realm, see the first
+                // note of <https://html.spec.whatwg.org/#realms-settings-objects-global-objects>
+                // and its following paragraph.
+                assert_not_equals(aTestElement.ownerDocument, null);
+                assert_not_equals(aTestElement.ownerDocument, undefined);
```

Comment:
Duplicate?

Reply:
Maybe. Due to the unclarity of such JS details, checking both explicitly seemed reasonable.
All other programming languages I'm aware of don't distinguish between `null` and `undefined`.